package.json

```diff
@@ -52,10 +55,12 @@
     "devDependencies": {
         "@changesets/changelog-github": "0.4.8",
         "@changesets/cli": "2.26.1",
+        "@swc/cli": "0.1.62",
```

Comment:
Not sure what alphabetized these but I like it, I wish that occurred by default for me :P

Reply:
I think it has something to do with the way I set up eslint with VSCode